api/end_device.proto

```diff
@@ -547,9 +547,13 @@ message EndDevice {
   EndDeviceAuthenticationCode claim_authentication_code = 46;
 
   // Skip decryption of uplink payloads and encryption of downlink payloads.
+  // This field is deprecated, use skip_payload_crypto_override instead.
   bool skip_payload_crypto = 51;
+  // Skip decryption of uplink payloads and encryption of downlink payloads.
+  // This field overrides the application-level setting.
+  google.protobuf.BoolValue skip_payload_crypto_override = 52;
```

Comment:
FYI, in MAC settings deprecated fields like these are renamed to `X_value`.
Since `skip_payload_crypto` is also there and also is an "override", I think we should just follow the same convention as MAC settings fields.

Reply:
In MAC settings we only allowed because it was internal API. This changes external API so we can't just go and rename fields.

Reply:
The new field needs to be added in `pkg/ttnpb/field_mask_validation.go`

Reply:
I'm not suggesting to rename any fields. To clarify, what I mean is this:

```suggestion
  google.protobuf.BoolValue skip_payload_crypto_value = 52;
```

Reply:
1. Why would having both `skip_payload_crypto` and `skip_payload_crypto_value` be more clear than naming what it is, i.e. an override?
2. We would get `SkipPayloadCryptoValue.Value` which stutters
3. There is no reason why we would religiously follow previous deprecated field name renames in future field names